tests/integration/clientside/test_clientside.py

```diff
@@ -0,0 +1,73 @@
+from selenium.webdriver.support.select import Select
+import time, os
```

Comment:
duplicate test files? Also may be nice to include in the test a second clientside function that takes 2 inputs.

Reply:
Yeah, was in the middle of :hocho: these, sorry I missed this one earlier!

Reply:
fixed in b85c6b7